payjoin-cli/src/app/v2.rs

```diff
@@ -162,7 +165,7 @@ impl App {
             let http = http_agent()?;
             let response = spawn_blocking(move || {
                 http.post(req.url.as_ref())
-                    .set("Content-Type", "text/plain")
+                    .set("content-type", "message/ohttp-req")
```

Comment:
the `content-type` is quite important in `payjoin` world, maybe we should consider add `RequestContentType` enum to the main crate with the available options. I believe these are `text/plain` and `message/ohttp-req`.

Reply:
I see two TODOs in this comment

1. define header key/value constants so these can't be misinterpreted during implementation
2. include headers in `req` from which to set them rather than set them manually

Beyond these TODOs, these are exactly the confusing implementation details that a `payjoin-io` crate could abstract.